src/GoogleAdWordsEventForwarder.js

```diff
@@ -199,10 +199,7 @@
         }
 
         function calculateJSHash(eventDataType, eventCategory, name) {
-            var preHash =
-                (eventDataType || '') + '' +
-                (eventCategory || '') + '' +
-                (name || '');
+            var preHash = '' + eventDataType + ('' + eventCategory) + '' + (name || '');
```

Comment:
Consider this:

```
var preHash = [
  eventDataType,
  eventCategory,
  name,
].join('')
```

Might make things more readable. and IIRC, `.join` should ignore undefined elements.

Reply:
def more readable.  thanks for the suggestion